### ISSUE TITLE
timestamp for files over last modified

### ISSUE THREAD

**Issue by cbates291** [awslabs/aws-js-s3-explorer#23]
Hello!

This project is awesome and looks great but i was wondering if there could either be a new column or a toggle between last modified and timestamp. The reason I ask, is because we would be using this to pull call recordings and we would need to compare timestamps to ensure we are pulling the correct calls, etc.

Thanks!

**Comment by john-aws** [comment]
Hi, I think what you're asking for is that as well as showing the last modified time in the current timeago format (e.g. '12 days ago', '3 months ago') we should also include an option to show a regular ISO 8601 timestamp (e.g. 2013-02-08 09:30:26.123).

Note that the current column sorting should sort correctly on the actual time value itself (not on the timeago text such as "3 days ago").

**Comment by cbates291** [comment]
yes! thats exactly what we would be looking for. The sorting makes sense and is nice but since agents could have 10-20 calls in the same day, it could still be a little hard to determine which call we are trying to pull (most of these get pulled because of some legal potential reasons), and it would just make it a lot easier to know exactly which item we want to download. So yes a simple option or toggle to show the timestamp would be perfect if possible.
